cpp/ql/src/semmle/code/cpp/exprs/Access.qll

```diff
@@ -18,7 +18,18 @@ abstract class Access extends Expr, NameQualifiableElement {
 }
 
 /**
- * A C/C++ enum constant access expression.
+ * A C/C++ enum constant access expression. For example the access to
+ * `MYENUMCONST1` in `myFunction` in the following code:
+ * ```
```

Comment:
I would back-tick `enum` here, and generally C/C++ keywords throughout.

Reply:
Done.